packages/client/modules/userDashboard/components/OrgBilling/OrgPlans.tsx

```diff
@@ -266,48 +264,48 @@ const OrgPlans = (props: Props) => {
           {showNudge && <LimitExceededWarning organizationRef={organization} />}
           <OrgStats organizationRef={organization} />
         </StyledRow>
-        <StyledRow>
+        <StyledRow className={'flex-col md:flex-row'}>
```

Comment:
+1 same here - I think we should add the styles to the styled component or use Tailwind with a `div`, but using a styled component with styles in the className feels confusing to me